docs/docs/dev_docs/contracts/syntax/functions.md

```diff
@@ -300,6 +300,14 @@ Each Aztec function has access to a [context](./context.mdx) object. This object
 As previously mentioned we use the kernel to pass information between circuits. This means that the return values of functions must also be passed to the kernel (where they can be later passed on to another function).
 We achieve this by pushing return values to the execution context, which we then pass to the kernel.
 
+**Making the contract's storage available**
+#include_code storage-example-context /yarn-project/noir-contracts/src/contracts/docs_example_contract/src/main.nr rust
```

Comment:
maybe also show how storage init works for public?

Reply:
In that case we should also show the expansion for a public function as the context uses different values too. I think people can infer that a different context goes in there.